molecule/test/unit/command/init/test_scenario.py

```diff
@@ -105,16 +105,9 @@ def test_execute_with_invalid_driver(
 ):
     _command_args['driver_name'] = 'ec3'
 
-    with pytest.raises(SystemExit) as e:
+    with pytest.raises(KeyError):
         _instance.execute()
 
-    assert 1 == e.value.code
```

Comment:
We should still be providing something to the user in the case of a `KeyError` instead of a stack trace!?

Reply:
@decentral1se You would be surprised but in this case the stack trace is more useful than the previous error message which was totally misleading: referring to missing folder, not inexistent driver.

Try it, that one is very easy to test from cli, before and after: `molecule init role --driver-name foo`.

Usually I am for clear error messages, but a nice error message here would require a wide range of changes. Should be done eventually but not part of this change.

Reply:
OK, sure. However, there is now an increasing list of "things for the future". Please start to mark them in the code! Add a TODO, XXX, whatever or create an issue. If you forget, nobody will remember to do these things until the users have to run into them.

Reply:
There where already there, the difference is that now we uncover them.

Reply:
I see no TODO in this change or added by you. So no, they are not there.